.github/workflows/build_BE.yml

```diff
@@ -15,38 +15,63 @@ jobs:
       max-parallel: 4
       matrix:
         python-version: [3.6, 3.7, 3.8]
+        node-version: ['10', '11', '12' ]
+    services:
+      postgres:
+        image: postgres:10.8
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: github_actions
+        ports:
+          - 5432:5432
+        # needed because the postgres container does not provide a healthcheck
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
 
     steps:
       - uses: actions/checkout@v2
+      
       - name: Set up Python ${{matrix.python-version}}
         uses: actions/setup-python@v1
         with:
-          python-version: ${{ matrix.python-version }}
+          python-version: ${{matrix.python-version}}
+      
+      - name: Set up Node ${{matrix.node-version}}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{matrix.node-version}}
 
-      - name: Install Dependencies
+      - name: Backend - Install Dependencies
         run: |
           python -m pip install --upgrade pip
           pip install -r requirements.txt
 
-      - name: Run Pytest
+      - name: Bug Finder - Pyflakes
+        uses: lgeiger/pyflakes-action@v1.0.1
+          
+      - name: Test + Coverage - Pytest
         run: pytest -v --cov=./ --cov-report=xml
 
+      - name: Lint - Flake8
```

Comment:
any way we can get this to generate a report to the root dir

Reply:
Easiest way I can think of doing this without messing with the action parameters is to have a silent action at the very end of the build that runs the flake8 command manually with --ignore flags. So it would be run twice. One ignoring Linting Errors and the other ignoring bug warnings.